### ISSUE TITLE
No shape/radius in Tile.Figure

### ISSUE THREAD

**Issue by talaviram** [novacbn/kahi-ui#130]
### Describe the Bug

I'm not sure if this is by design and I might've missed something.
But it's also not stated in the the changes between `0.5.7 -> 0.6.x`.

In previous versions I had:
```
<Tile.Figure shape="pill">
...
</Tile.Figure>
```

But now there's no simple way without adding extra div and a non Title originated Figure.

### Reproduction

n/a

### System Info

```shell
Tested with macOS 12.2 / Firefox 98.0.1
```

**Comment by novacbn** [comment]
Should be fixed in the `0.6.4` release. I'll update the documentation later, but it's the same API as [Figure](https://kahi-ui.nbn.dev/docs/embedded/figure).

**Comment by talaviram** [comment]
updated to `0.6.4` and it seems this is resolved.
thanks!
